src/main/java/com/cburch/logisim/gui/prefs/WindowOptions.java

```diff
@@ -184,11 +187,11 @@ public WindowOptions(PreferencesFrame window) {
     initThemePreviewer();
 
     setLayout(new TableLayout(1));
-    final var but = new JButton();
-    but.addActionListener(listener);
-    but.setActionCommand(cmdResetWindowLayout);
-    but.setText(S.get("windowToolbarReset"));
-    add(but);
+    ResetWindowLayoutButton = new JButton();
+    ResetWindowLayoutButton.addActionListener(listener);
+    ResetWindowLayoutButton.setActionCommand(cmdResetWindowLayout);
+    ResetWindowLayoutButton.setText(S.get("windowToolbarReset"));
+    add(ResetWindowLayoutButton);
```

Comment:
Please follow the naming convention that variable names should start with a lowercase char.

Reply:
Normally, this is corrected